async-query-core/src/main/antlr/SqlBaseLexer.g4

```diff
@@ -276,13 +277,15 @@ INTO: 'INTO';
 INVOKER: 'INVOKER';
 IS: 'IS';
 ITEMS: 'ITEMS';
+ITERATE: 'ITERATE';
```

Comment:
revert it if not related to this PR.

Reply:
reverted g4 files and created issue to disable auto sync https://github.com/opensearch-project/sql/issues/3022